code/modules/core_implant/cruciform/machinery/obelisk.dm

```diff
@@ -2,7 +2,9 @@ GLOBAL_LIST_EMPTY(all_obelisk)
 
 /obj/machinery/power/nt_obelisk
 	name = "NeoTheology's obelisk"
-	desc = "The obelisk."
+	desc = "An obelisk of the Church which grants protection and strength to those who bear cruciforms."
+	description_info = "Obelisks help provide observation for the Eye, alongside granting faster sanity regeneration to NT faithful. In addition, when \"active\", they will repel weeds and damage hostile fauna such as roaches within range."
```

Comment:
Do not use abbreviations in descriptions

Reply:
Fixed it